bin/_mocha

```diff
@@ -524,10 +526,10 @@ mocha.ui(program.ui);
 
 const args = program.args;
 
-// default files to test/*.{js,coffee}
+// sets directory for test files with default to test/*.{js,coffee}
```

Comment:
Change this comment (since it's wrong anyway)
`// Set default directory for test files`